stepler/nova/steps/servers.py

```diff
@@ -141,7 +141,9 @@ def attach_floating_ip(self, server, floating_ip, check=True):
 
         if check:
             floating_ips = self.get_ips(server, 'floating').keys()
-            floating_ip.ip in floating_ips
+            assert floating_ip.ip in floating_ips, (
```

Comment:
@AlexanderKoryagin, @TimurNurlygayanov, @mshalamov, guys, how about to use pyhamcrest? Seems, to compose assert message each time looks isn't good. For ex:

```python
floating_ip = '172.0.0.1'
floating_ips = ['172.0.0.2', '172.0.0.3']

assert floating_ip in floating_ips, (
    "Floating IP '{0}' not in a list of server's IPs '{1}'.".format(
        floating_ip, floating_ips))

>>> AssertionError: Floating IP '172.0.0.1' not in a list of server's IPs '['172.0.0.2', '172.0.0.3']'.
```

```python
from hamcrest import assert_that, has_item

assert_that(floating_ips, has_item(floating_ip))

>>> AssertionError: 
Expected: a sequence containing '172.0.0.1'
     but: was <['172.0.0.2', '172.0.0.3']>

```